test/e2e-image/e2e.sh

```diff
@@ -39,18 +39,34 @@ ginkgo "${ginkgo_args[@]}"               \
   -focus="${FOCUS}"                  \
   -skip="\[Serial\]|\[MemoryLeak\]"  \
   -nodes="${E2E_NODES}" \
+  --junit-report=report_e2e_test_suite.xml \
   /e2e.test
+echo "creating configMap report1 from the report file now.."
+kubectl create cm report1 --from-file report_e2e_test_suite.xml
```

Comment:
I know this is an initial PR, but if possible:
* apply gz over report.xml so we don’t hit k8s/etcd limit of 2mb per cm
* encode the gz as base64

then you revert on the other side